File: problem2.cpp
#include <iostream>
using namespace std;

int main() {
	int first = 1;
	int second = 1;
	int sum = 0;
	int num;
	while (second < 4000000) {
		num = first + second;
		if (num % 2 == 0) {
			sum = sum + num;
		}
		first = second;
		second = num;
	}
	cout << sum << endl;
}

File: solutions/python/problem2.py
def main():
	x=2
	y=1
	num = 0
	while x < 4000000:
		print(x , y)
		temp = x		
		if x % 2 == 0: 
			num += x		
		x = x + y
		y = temp
	print(num)
	
main()